app/javascript/controllers/dynamic_form_controller.js

```diff
@@ -1,8 +1,6 @@
 import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
-  static targets = ["link"]
```

Comment:
I think maybe you didn't mean to remove this?  We still need it.

Reply:
Yep. My mistake. Fixed.